common/source/docs/common-auxiliary-functions.rst

```diff
@@ -0,0 +1,633 @@
++----------------------+----------------------------+----------+---------+---------+
+|        22            | Parachute Release          |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        23            | Parachute 3 Pos Switch     |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        24            | Reset Auto Mission to start|    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        25            | Attitude Controller FF     |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        26            | Attitude Controller AccLim |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        27            | Retract Mount              |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        28            | Relay 1 On/off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        29            | Landing Gear               |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        30            | Lost Vehicle Sound         |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        31            | Motor Emergency Stop       |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        32            | Motor Interlock            |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        33            | Brake                      |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        34            | Relay 2 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        35            | Relay 3 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        36            | Relay 4 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        37            | THROW mode                 |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        38            | Avoid ADSB                 |    X     |         |         |
```

Comment:
We should probably append "mode" to "Avoid ADSB" and maybe it should be all caps?

Reply:
Same comment as FLIP...the wiki currently says it should be set as a flight mode, shouldn't we remove it from the FLTMODEx param listing also?

Reply:
It really is a flight mode although it's true that users shouldn't really need to set it up on a flight mode switch nor an RC channel.  On the otherhand it won't do any harm either, the vehicle will just sit there loitering in place and not accepting any pilot input.

We need to keep the mode in the FLTMODEx parameter listing or else the GCSs won't know what string to display to the users when the vehicle does go into this mode.  It would appear as "Unknown".

My guess is that AvoidADSB was added to the RC_Options list as part of a review of all missing modes.  It shouldn't really have been added.  We could remove it.

Reply:
per conv changed name...will PR a code change for RC_Channel tomorrow to match ADSB Avoidance Enable